spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -748,7 +748,7 @@ public void open() {
       cursor = completionText.length();
     }
     if (Utils.isScala2_10()) {
-      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completor");
+      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completer");
```

Comment:
More `completor` in here :)[spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L79](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L79) & [spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L81](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/DepInterpreter.java#L81)